### ISSUE TITLE
SSL CA bundle not found

### ISSUE THREAD

**Issue by joshuap** [honeybadger-io/honeybadger-laravel#88]
A customer reported getting this message when testing Honeybadger in production:

```
Write HONEYBADGER_API_KEY to .env: ✔
Write HONEYBADGER_API_KEY placeholder to .env.example: ✔
Write HONEYBADGER_VERIFY_SSL placeholder to .env.example: ✔

There was an error sending the payload to Honeybadger: SSL CA bundle not found:
```

@shalvah (or anyone else)—any ideas on this? We bundle a ca-cert in our honeybadger-ruby gem; is that something we could provide w/ PHP, or can it be configured on the server?

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_3jxnl)

**Comment by jonzenor** [comment]
Just some info on my environment:
 - Linode Server
 - Managed by Laravel Forge (Forge did the install & build)
 - Running php 8.x
 - Laravel 8.x

This same version of PHP and Laravel are used in my test environment where the Honeybadger artisan command works, but my test environment does not use SSL.

**Comment by shalvah** [comment]
Hmmm. Guzzle provides a [`verify` option](https://docs.guzzlephp.org/en/stable/request-options.html#verify) that lets you specify a custom CA bundle (although I've only ever seen .pem files used, but the Ruby lib uses .crt, even though the contents appear to be equivalent). 

@jonzenor you can download the `ca-bundle.crt` file from the Ruby library [here](https://raw.githubusercontent.com/honeybadger-io/honeybadger-ruby/2d24a36f5f560c4234f22f59153c80f32e4ce177/resources/ca-bundle.crt), then place it somewhere in your repo and pass the absolute path to it to the `client.verify` option in your `honeybadger.php` and try again. Tried it and there were no issues.

**Comment by shalvah** [comment]
Be sure to `config:clear` after editing the config file.

**Comment by jonzenor** [comment]
It turns out that all I had to do was set HONEYBADGER_VERIFY_SSL to true in my .env file... In my local (dev) environment only the HONEYBADGER_API_KEY was added to .env, but both verify_ssl and api_key were added to my .env.example. So when I deployed to development for the first time, the verify_ssl config setting was added to .env but it was left blank. I didn't see any instructions telling me to set it, so I just left it at its default setting.

So I changed this:
``HONEYBADGER_VERIFY_SSL=``

To this:
``HONEYBADGER_VERIFY_SSL=true``

Now it works.
